### PR TITLE
Address several deprecation warnings

### PR DESCRIPTION
**Pull request by Eitot**
This fixes/suppresses several deprecation warnings.

I started working on this warning too:
Screenshot 2020-11-15 at 18 36 52

However, DownloadManager implements `WebDownloadDelegate` which is a subprotocol of `NSURLDownloadDelegate`. This means that the TabbedWebView has a dependency on `NSURLDownload` too.

@TAKeanice: Have you come across any functions in WebKit that let us handle the download manager for file downloads?

### CODE REVIEW

**Comment by TAKeanice** [comment]
No, that is one of the TODOs I also mentioned in the wiki article

**Comment by josh64x2** [comment]
Travis build is currently failing due to back-end issues.

**Comment by Eitot** [comment]
I think [`WKNavigationResponse`](https://developer.apple.com/documentation/webkit/wknavigationresponse) is a likely candidate, implemented via WKNavigationDelegate, in particular this method: [`webView:decidePolicyForNavigationResponse:decisionHandler:`](https://developer.apple.com/documentation/webkit/wknavigationdelegate/1455643-webview)

File: Vienna/Sources/Shared/ImageAndTextCell.m [hunk 1]
```diff
@@ -325,32 +325,26 @@ -(void)selectWithFrame:(NSRect)aRect inView:(NSView *)controlView editor:(NSText
...
 
--(NSArray*)accessibilityAttributeNames
-{
-    static NSArray * attributes = nil;
-    if (!attributes)
-    {
-        NSSet * set = [NSSet setWithArray:[super accessibilityAttributeNames]];
-        attributes = [set setByAddingObject:NSAccessibilityDescriptionAttribute].allObjects;
+- (NSString *)accessibilityLabel {
+    NSMutableArray *bits = [NSMutableArray arrayWithCapacity:3];
```
Comment (josh64x2) [inline Vienna/Sources/Shared/ImageAndTextCell.m:329]: is `bits` the best name to use here? How about something like `accessibilityLabelData` or similar?

Comment (Eitot) [reply Vienna/Sources/Shared/ImageAndTextCell.m:329]: I didn’t change this part of the code. 😅

Comment (josh64x2) [reply Vienna/Sources/Shared/ImageAndTextCell.m:329]: heh OK

**Review by josh64x2 — APPROVED** [review]

**Pull request merged**